wiki/Guides/Compressing_files/en.md

```diff
@@ -80,42 +92,74 @@ ffmpeg -i input -c:v libx264 -crf 20 -preset slower -profile:v high -vf scale=-1
 
 ## Audio
 
-Audio encoded in either MP3 or OGG (Vorbis) formats are supported with `.mp3` and `.ogg` file extensions, respectively. Other formats such as AAC and OGG (Opus) and any lossless formats are not supported by the game client.
+**Audio encoded in either MP3 or OGG (Vorbis) formats are supported with `.mp3` and `.ogg` file extensions, respectively.** Other formats such as AAC and OGG (Opus) and any lossless formats, except for uncompressed PCM audio with the `.wav` file extension for hitsounds, are currently not supported by the game client.
```

Comment:
1) audio is singular
2) the second sentence may be significantly reduced for massive comprehension gain
```suggestion
**Audio encoded in either MP3 or OGG (Vorbis) formats is supported with `.mp3` and `.ogg` file extensions, respectively.** Other formats are currently not supported (except for audio with the `.wav` file extension for hitsounds).
```